readalongs/cli.py

```diff
@@ -66,116 +73,132 @@ def cli():
-    if kwargs['output_xhtml']:
-        tokenized_xml_path = '%s.xhtml' % output_base
+        os.mkdir(kwargs["output_base"])
+    output_base = os.path.join(
+        kwargs["output_base"], os.path.basename(kwargs["output_base"])
+    )
+    if kwargs["debug"]:
+        LOGGER.setLevel("DEBUG")
+    if kwargs["text_input"]:
+        if not kwargs["language"]:
+            LOGGER.warn(
+                f"No input language provided, using undetermined mapping")
+        tempfile, kwargs["inputfile"] = create_input_tei(
+            kwargs["inputfile"],
+            text_language=kwargs["language"],
+            save_temps=(output_base if kwargs["save_temps"] else None),
+        )
+    if kwargs["output_xhtml"]:
+        tokenized_xml_path = "%s.xhtml" % output_base
     else:
-        _, input_ext = os.path.splitext(kwargs['inputfile'])
-        tokenized_xml_path = '%s%s' % (output_base, input_ext)
-    if os.path.exists(tokenized_xml_path) and not kwargs['force_overwrite']:
-        raise click.BadParameter("Output file %s exists already, did you mean to do that?"
-                                 % tokenized_xml_path)
-    smil_path = output_base + '.smil'
-    if os.path.exists(smil_path) and not kwargs['force_overwrite']:
-        raise click.BadParameter("Output file %s exists already, did you mean to do that?"
-                                 % smil_path)
-    _, wav_ext = os.path.splitext(kwargs['wavfile'])
-    wav_path = output_base + wav_ext
-    if os.path.exists(wav_path) and not kwargs['force_overwrite']:
-        raise click.BadParameter("Output file %s exists already, did you mean to do that?"
-                                 % wav_path)
+        _, input_ext = os.path.splitext(kwargs["inputfile"])
+        tokenized_xml_path = "%s%s" % (output_base, input_ext)
+    if os.path.exists(tokenized_xml_path) and not kwargs["force_overwrite"]:
+        raise click.BadParameter(
+            "Output file %s exists already, did you mean to do that?"
+            % tokenized_xml_path
+        )
+    smil_path = output_base + ".smil"
+    if os.path.exists(smil_path) and not kwargs["force_overwrite"]:
+        raise click.BadParameter(
+            "Output file %s exists already, did you mean to do that?" % smil_path
+        )
+    _, audio_ext = os.path.splitext(kwargs["audiofile"])
+    audio_path = output_base + audio_ext
+    if os.path.exists(audio_path) and not kwargs["force_overwrite"]:
+        raise click.BadParameter(
+            "Output file %s exists already, did you mean to do that?" % audio_path
+        )
     unit = kwargs.get("unit", "w")
     bare = kwargs.get("bare", False)
-    if not unit:     # .get() above should handle this but apparently the way kwargs is implemented
-        unit = "w"   # unit could still be None here.
+    if (
+        not unit
+    ):  # .get() above should handle this but apparently the way kwargs is implemented
```

Comment:
OK, I really don't like this! Why break such a short statement over three lines?

Reply:
I'm not sure how this happened!

Reply:
I bet Black split it because the original line was longer that 88 characters when you consider the length of the comment.
I'm willing to accept such weird things if we're agreeing to an automated formatting tool.